packages/csv-parser-discount-code/package.json

```diff
@@ -28,7 +28,7 @@
     "lib"
   ],
   "scripts": {
-    "build": "babel src --out-dir lib"
+    "build": "babel src --out-dir lib --config-file '../../babel.config.js'"
```

Comment:
Other ideas would be appreciated here. I do prefer this over having a config file inside of every package but it's definitely not ideal.

Reply:
I think it's fine like this

Reply:
Yes this is fine. To my knowledge the idea of a babel.config over an rc without being breaking is to be more explicit. Meaning you pass it, you import it etc.